docker/nginx/libs/skynet/account.lua

```diff
@@ -78,7 +78,8 @@ function _M.has_subscription()
 end
 
 function _M.is_auth_required()
-    return os.getenv("ACCOUNTS_LIMIT_ACCESS") == "authenticated"
+    -- authentication is required if mode is set to "authenticated" or "subscription"
+    return os.getenv("ACCOUNTS_LIMIT_ACCESS") == "authenticated" or _M.is_subscription_required()
```

Comment:
nit: I assume `subscription` implies a paid tier? It would be nice to add a comment to that effect.

Reply:
added in https://github.com/SkynetLabs/skynet-webportal/pull/1852/commits/d0de18fcd0270a5bda3835048b83aba6c21310c4